Documentation/Diagnostics/PH2155.md

```diff
@@ -34,20 +34,147 @@ Packages with unacceptable licenses, particularly copyleft licenses like GPL, ca
 <PackageReference Include="SomeGplPackage" Version="1.0.0" />
 ```
 
+### Diagnostic message
```

Comment:
Let's add a disclaimer somewhere in this md file that this analyzer is only meant to shift-left the developer experience when adding new packages, and should no way be construed to be a replacement for commercial products devoted to this topic. I.e., do not rely on this analyzer as a source of truth, and the author is neither accountable nor responsible for mistakes. Moreover, this analyzer only examines nuget package references; there are several other ways to reference 3rd party modules, such as old-fashioned direct references and COM references.